src/sentry/options/defaults.py

```diff
@@ -396,6 +396,11 @@
 # 0 default value means don't limit.
 # The limits are per second, applied across a window. The actual quota is then `window * limit`.
 # If the limit is negative, then it means completely blocked.
+register("project-abuse-quota.window", type=Int, default=10, flags=FLAG_PRIORITIZE_DISK)
+register("project-abuse-quota.error-limit", type=Int, default=0, flags=FLAG_PRIORITIZE_DISK)
+register("project-abuse-quota.transaction-limit", type=Int, default=0, flags=FLAG_PRIORITIZE_DISK)
+register("project-abuse-quota.attachment-limit", type=Int, default=0, flags=FLAG_PRIORITIZE_DISK)
+register("project-abuse-quota.session-limit", type=Int, default=0, flags=FLAG_PRIORITIZE_DISK)
```

Comment:
Re-added these as the removal was causing errors:
```
Traceback (most recent call last):
  File "/usr/src/sentry/src/sentry/api/base.py", line 139, in handle_exception
    response = super().handle_exception(exc)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 465, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 476, in raise_uncaught_exception
    raise exc
  File "/usr/src/sentry/src/sentry/api/base.py", line 254, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/src/sentry/src/sentry/api/endpoints/relay/project_configs.py", line 35, in post
    return self._post(request)
  File "/usr/src/sentry/src/sentry/api/endpoints/relay/project_configs.py", line 50, in _post
    return self._post_by_key(
  File "/usr/src/sentry/src/sentry/api/endpoints/relay/project_configs.py", line 128, in _post_by_key
    project_config = config.get_project_config(
  File "/usr/src/sentry/src/sentry/relay/config/__init__.py", line 147, in get_project_config
    public_keys = get_public_key_configs(project, full_config, project_keys=project_keys)
  File "/usr/src/sentry/src/sentry/relay/config/__init__.py", line 81, in get_public_key_configs
    q.to_json_legacy() for q in quotas.get_quotas(project, key=project_key)
  File "/usr/src/sentry/src/sentry/utils/services.py", line 127, in <lambda>
    context[key] = (lambda f: lambda *a, **k: getattr(self, f)(*a, **k))(key)
  File "/usr/src/sentry/src/sentry/quotas/redis.py", line 58, in get_quotas
    results = [*self.get_project_abuse_quotas(project.organization)]
  File "/usr/src/sentry/src/sentry/quotas/base.py", line 337, in get_project_abuse_quotas
    abuse_window = options.get("project-abuse-quota.window")
  File "/usr/src/sentry/src/sentry/options/manager.py", line 128, in get
    opt = self.lookup_key(key)
  File "/usr/src/sentry/src/sentry/options/manager.py", line 102, in lookup_key
    raise UnknownOption(key)
sentry.options.manager.UnknownOption: 'project-abuse-quota.window'
```

Reply:
Yup, sorry, this was my mistake. Got confused with options vs org optiuons.